# ROS2 parameter format for Reference Costmap Generator
reference_costmap_generator:
  ros__parameters:
    ref_path_topic: /plan
    map_topic: /map
    goal_pose_marker_topic: /goal_pose_marker
    distance_error_map_topic: /distance_error_map
    ref_yaw_map_topic: /ref_yaw_map
    map_resolution_scale: 1.0 # ex.) 10.0 means 10 times as rough as the original map resolution
